TombEngine/Game/effects/spark.cpp

```diff
@@ -172,6 +172,10 @@ namespace TEN::Effects::Spark
 		spark.friction = 34;
 		spark.maxYvel = 0;
 		spark.gravity = 0;
+		spark.scalar = 2;
```

Comment:
This makes all three values the same (GenerateInt is called once, not thrice). Is this intended?

Reply:
yes, because some of the sparks would look ugly because they would be stretched then